module/zcommon/zfs_prop.c

```diff
@@ -245,6 +245,14 @@ zfs_prop_init(void)
 		{ NULL }
 	};
 
+	static zprop_index_t volmode_table[] = {
+		{ "default",	ZFS_VOLMODE_DEFAULT },
+		{ "geom",	ZFS_VOLMODE_GEOM },
```

Comment:
Do you mean keeping `geom` and adding `all` as a 5th value in `volmode_table` or replacing `geom` with `all`?
I'm ok with this (seems a small change) but wouldn't this be confusing for users coming from/working with FreeBSD?